packages/editor-ui/src/components/WorkflowActivator.test.ts

```diff
@@ -0,0 +1,80 @@
+import { describe, it, expect, vi } from 'vitest';
+import WorkflowActivator from '@/components/WorkflowActivator.vue';
+import userEvent from '@testing-library/user-event';
+
+import { useWorkflowsStore } from '@/stores/workflows.store';
+
+import { createTestingPinia } from '@pinia/testing';
+import { createComponentRenderer } from '@/__tests__/render';
+import { mockedStore } from '@/__tests__/utils';
+import { EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE } from '@/constants';
+
+const renderComponent = createComponentRenderer(WorkflowActivator);
+let mockWorkflowsStore: ReturnType<typeof mockedStore<typeof useWorkflowsStore>>;
+
+describe('WorkflowActivator', () => {
+	beforeEach(() => {
+		createTestingPinia();
+
+		mockWorkflowsStore = mockedStore(useWorkflowsStore);
+	});
+
+	afterEach(() => {
+		vi.clearAllMocks();
+	});
+
+	it('it renders correctly', () => {
```

Comment:
Nit: No need to repeat `it` in the test name
```suggestion
	it('renders correctly', () => {
```